recipes/simplification/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "simplification" %}
+{% set version = "0.5.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/urschrei/simplification/archive/v{{ version }}.tar.gz
```

Comment:
Any reason not to use the tarball from PyPI?

Reply:
Couldn't find a tarball on the PyPI - the 'Download' section on the Simplfication PyPI points to GitHub:
https://github.com/urschrei/simplification/tarball/v0.5.7

Reply:
Oh, there are only wheels on PyPI. That's...strange. Ok, I suppose GH is the way to go then.